src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/cumulative-metrics-transition-guide.mdx

```diff
@@ -0,0 +1,151 @@
+
+If you were a customer using cumulative metrics before March 14, you should check and potentially modify existing queries and alerts that may be relying on those metrics. The sections below will teach you: 
+
+* How to know if your account is impacted
+* How to prepare for the EoL
+* How to preview the `cumulativeCount` functionality 
+
+<Callout variant="important">
+Non-monotonic cumulative sums will **not** be affected by this upgrade.
+</Callout>
+
+## Check your account for impact [#impact]
+
+If your account was sending OTLP cumulative sum metrics to New Relic before March 14, 2023, then that account will be impacted by the EoL on June 13, 2023. 
+
+To check if your account is impacted, run the following NRQL query:
+
+```
+FROM NrIntegrationError SELECT count(*) 
+WHERE newRelicFeature = 'cumulativeSumAsGaugeEoL' 
+SINCE 24 hours ago
+```
+
+The query will return results if you've reported metrics impacted by this EoL in the last 24 hours. 
+
+To investigate further, you can view the event message to see which metrics are affected:
+
+```
+FROM NrIntegrationError SELECT metricName 
+WHERE newRelicFeature = 'cumulativeSumAsGaugeEoL' 
+LIMIT MAX SINCE 24 hours ago
+```
+
+Any monitoring or alerting involving these metrics will need to be reviewed and updated prior to the EoL date. The next section contains suggested actions for impacted metrics. 
+
+## What to do if you're impacted [#what-to-do]
+
+We recommend impacted accounts to check and potentially modify existing NRQL-generated charts and alerts that target OTLP cumulative sum metrics. Our [OpenTelemetry best practices](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-best-practices-metrics#query) outline the differences in querying between a `gauge` and a `cumulativeCount` metric type. Namely, you'll be able to target either the delta value (by default) or the cumulative value by selecting the cumulative field. 
+
+Below we focus on updating [alert conditions](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions), because alerts are generally more important than charts. For your custom charts, you can either:
+
+* Take the same general approach we describe for alerts: create a new chart with an updated query and erase the old chart after the EoL, OR
+* Wait until after the EoL and replace the queries
+
+## Suggested action: Create new versions of alerts [#alerts]
+
+Here are some examples of queries that might be used in alerts. We'll give you the original query, then a `gauge` version of the query, and a `cumulativeCount` version of the query. You should replace your existing alert with the `gauge` version of the query, and create a duplicate alert condition that uses the `cumulativeCount` version. That way, when we transition your account to `cumulativeCount` on June 16, that alert condition will be available when the old query no longer works. 
```

Comment:
Here are some questions:

* How about calling "original query" something like the generic query or starting query?
* Why does this say that you should replace your existing alert with the `gauge` version of the query? Weren't existing users already using gauges in their queries since monotonic cumulative sums used to be converted to gauges? I was just expecting to be told to add a duplicate alert condition that uses cumulativeCount.

Reply:
@tclayton-newr question for you: why do we tell them to replace the existing one with the `gauge`? I don't really get that either; wouldn't it work remaining as is? Why does the existing query need to change?

Reply:
That's a great question! The existing query needs to target the `gauge` type specifically because if the account updates to use the `cumulativeCount` type, the default value returned by the query will change. For example, consider a metric reporting uptime (in seconds) of a server every second (a counter increasing by 60 every minute) and we'll assume we're at 120 seconds of uptime. 
Then the `gauge` query 
```
FROM Metric SELECT latest(<metricName>)
```
would return 120 since (prior to this EoL) that was the best value we could return; the latest cumulative value the account sent us. 

However, the `cumulativeCount` version of the same query would now be targeting a **delta** value instead of the cumulative value. So it would return a value of 1, because the delta rate of change is likely a value of 1 every second. While this example is somewhat contrived (a customer likely isn't alerting based on uptime of their servers) it does show how a query can change from reporting an ever-increasing counter to just the most recent delta value, which can be two very different results.